.editorconfig

```diff
@@ -0,0 +1,8 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Please remove this file. First of all, it has nothing to do with the tests and the content doesn't fit with what exists in this repo (e.g. tabs for indentation).

Reply:
Fixed.